src/components/Footer/Footer.js

```diff
@@ -7,47 +7,87 @@
  * LICENSE.txt file in the root directory of this source tree.
  */
 
-import React from 'react';
+import React, { Component, PropTypes } from 'react';
 import withStyles from 'isomorphic-style-loader/lib/withStyles';
 // Import custom footer styles
 import s from './Footer.css';
 
+import FontIcon from 'material-ui/FontIcon';
+import {BottomNavigation, BottomNavigationItem} from 'material-ui/BottomNavigation';
+import Paper from 'material-ui/Paper';
+import IconLocationOn from 'material-ui/svg-icons/communication/location-on';
+
+const nearbyIcon = <IconLocationOn />;
+
 class Footer extends React.Component {
+  static propTypes = {
+    isLoggedIn: PropTypes.bool,
```

Comment:
Can you add 'isRequired' to the prop type here? We don't have to write a test to validate 'isLoggedIn' then.